packages/manager/cypress/e2e/databases/update-database.spec.ts

```diff
@@ -140,223 +143,234 @@ const resetRootPassword = () => {
-   * - Confirms that users cannot change access controls for provisioning DBs.
-   * - Confirms that users cannot reset root passwords for provisioning DBs.
-   * - Confirms that users cannot change maintenance schedules for provisioning DBs.
-   */
-  it('Cannot update database clusters while they are provisioning', () => {
-    const initialLabel = randomLabel();
-    const updateAttemptLabel = randomLabel();
-    const allowedIp = randomIp();
-    const engineType = randomItem(databaseTypes);
-    const database = databaseFactory.build({
-      id: randomNumber(1, 1000),
-      type: 'g6-nanode-1',
-      label: initialLabel,
-      region: randomItem(regions),
-      engine: engineType,
-      status: 'provisioning',
-      allow_list: [allowedIp],
-      hosts: {
-        primary: undefined,
-        secondary: undefined,
-      },
-    });
-
-    const errorMessage =
-      'Your database is provisioning; please wait until provisioning is complete to perform this operation.';
-    const hostnameRegex = /your hostnames? will appear here once (it is|they are) available./i;
-
-    mockGetDatabase(database).as('getDatabase');
-
-    mockUpdateProvisioningDatabase(
-      database.id,
-      database.engine,
-      errorMessage
-    ).as('updateDatabase');
-
-    mockResetPasswordProvisioningDatabase(
-      database.id,
-      database.engine,
-      errorMessage
-    ).as('resetRootPassword');
-
-    cy.visitWithLogin(`/databases/${database.engine}/${database.id}`);
-    cy.wait('@getDatabase');
-
-    // Cannot update database label.
-    updateDatabaseLabel(initialLabel, updateAttemptLabel);
-    cy.wait('@updateDatabase');
-    cy.findByText(errorMessage).should('be.visible');
-    cy.get('[data-qa-cancel-edit="true"]')
-      .should('be.visible')
-      .should('be.enabled')
-      .click();
-
-    cy.get('[data-qa-connection-details]').within(() => {
-      // DBaaS hostnames are not available until database/cluster has provisioned.
-      cy.findByText(hostnameRegex).should('be.visible');
-
-      // DBaaS passwords cannot be revealed until database/cluster has provisioned.
-      cy.findByText('Show')
-        .closest('button')
-        .should('be.visible')
-        .should('be.disabled');
-    });
-
-    // Cannot add or remove allowed IPs before database/cluster has provisioned.
-    removeAllowedIp(allowedIp);
-    cy.wait('@updateDatabase');
-    ui.dialog
-      .findByTitle(`Remove IP Address ${allowedIp}`)
-      .should('be.visible')
-      .within(() => {
-        cy.findByText(errorMessage).should('be.visible');
-        ui.buttonGroup.findButtonByTitle('Cancel').should('be.visible').click();
+  databaseConfigurations.forEach(
+    (configuration: databaseClusterConfiguration) => {
+      it(`creates a ${configuration.linodeType} ${configuration.engine} v${configuration.version}.x ${configuration.clusterSize}-node cluster at ${configuration.region}`, () => {
```

Comment:
Should `creates` be `updates` here?

```suggestion
      it(`updates a ${configuration.linodeType} ${configuration.engine} v${configuration.version}.x ${configuration.clusterSize}-node cluster at ${configuration.region}`, () => {
```